src/app/components/comment/comment.component.ts

```diff
@@ -12,12 +12,14 @@ import UserMiniDTO from 'src/app/models/UserMiniDTO';
   styleUrls: ['./comment.component.css']
 })
 export class CommentComponent implements OnInit {
+  replyState: boolean = false;
```

Comment:
Sonar lint doesn't like this line. Just remove the ": boolean"

Reply:
Whoa this is a neat comment

Reply:
I'll grab this bad boy